### ISSUE TITLE
Request i?max i?sum API

### ISSUE THREAD

**Issue by tianzhipengfei** [xianyi/OpenBLAS#2068]
I am wondering all API are listed in cblas.h? In my opinion, i?max, i?sum are very common operations, but I cannot find them in cblas.h but in f77blas.h. So I don't know whether I didn't find them or OpenBLAS doesn't support them currently.

**Comment by brada4** [comment]
In `cblas.h` all functions are prepended with `cblas_`
```
float  cblas_sasum (OPENBLAS_CONST blasint n, OPENBLAS_CONST float  *x, OPENBLAS_CONST blasint incx);
double cblas_dasum (OPENBLAS_CONST blasint n, OPENBLAS_CONST double *x, OPENBLAS_CONST blasint incx);
float  cblas_scasum(OPENBLAS_CONST blasint n, OPENBLAS_CONST void  *x, OPENBLAS_CONST blasint incx);
double cblas_dzasum(OPENBLAS_CONST blasint n, OPENBLAS_CONST void *x, OPENBLAS_CONST blasint incx);
CBLAS_INDEX cblas_isamax(OPENBLAS_CONST blasint n, OPENBLAS_CONST float  *x, OPENBLAS_CONST blasint incx);
CBLAS_INDEX cblas_idamax(OPENBLAS_CONST blasint n, OPENBLAS_CONST double *x, OPENBLAS_CONST blasint incx);
CBLAS_INDEX cblas_icamax(OPENBLAS_CONST blasint n, OPENBLAS_CONST void  *x, OPENBLAS_CONST blasint incx);
CBLAS_INDEX cblas_izamax(OPENBLAS_CONST blasint n, OPENBLAS_CONST void *x, OPENBLAS_CONST blasint incx);
```

**Comment by martin-frbg** [comment]
This was reported (and fixed) for cblas_i?amin in #1637, somehow I managed to miss that cblas_i?max and cblas_i?min have the same problem. I do not think OpenBLAS has a non-absolute sum, though that should be trivial to implement. (Not sure if this should be named i?sum or simply ?sum - or even both)

**Comment by tianzhipengfei** [comment]
> This was reported (and fixed) for cblas_i?amin in #1637, somehow I managed to miss that cblas_i?max and cblas_i?min have the same problem. I do not think OpenBLAS has a non-absolute sum, though that should be trivial to implement. (Not sure if this should be named i?sum or simply ?sum - or even both)

Thanks a lot. Because non-absolute sum operation is very common in neural network, is it possible that OpenBLAS (in C Language) will support non-absolute operation such as sum, max, min? 

(BTW, maybe ?sum is better, i?sum is a mistake. Sorry for that)
